cms/static/js/views/group_configuration_item.js

```diff
@@ -1,77 +1,42 @@
+/**
+ * This class defines an controller view for content experiment group configurations.
+ * It renders an editor view or a details view depending on the state
+ * of the underlying model.
+ * It is expected to be backed by a Group model.
+ */
 define([
-    'js/views/baseview', 'jquery', "gettext", 'js/views/group_configuration_details',
-    'js/views/group_configuration_edit', "js/views/utils/view_utils"
+    'js/views/list_item', 'js/views/group_configuration_details', 'js/views/group_configuration_editor'
 ], function(
-    BaseView, $, gettext, GroupConfigurationDetails, GroupConfigurationEdit, ViewUtils
+    ListItemView, GroupConfigurationDetailsView, GroupConfigurationEditorView
 ) {
     'use strict';
-    var GroupConfigurationsItem = BaseView.extend({
+
+    var GroupConfigurationItemView = ListItemView.extend({
+        events: {
+            'click .delete': 'deleteItem'
+        },
+
         tagName: 'section',
+
         attributes: function () {
             return {
                 'id': this.model.get('id'),
                 'tabindex': -1
             };
         },
-        events: {
-            'click .delete': 'deleteConfiguration'
-        },
 
-        className: function () {
-            var index = this.model.collection.indexOf(this.model);
+        itemDisplayName: 'Group Configuration',
```

Comment:
Good spot. `itemDisplayName` is used by the `ListItemView` to render a delete confirmation modal. The lower-cased string you're referring to is defined as the `itemCategoryDisplayName` on the `ListView`s. Since they both represent the same thing, I'm going to see if they can be refactored. @catong could you please confirm the casing for these copies that appear in the add button and the remove dialogue?